### ISSUE TITLE
Bug: Projects are not saved as ".side", but as ".txt" in Chrome 78

### ISSUE THREAD

**Issue by lepantella** [SeleniumHQ/selenium-ide#861]
## 🐛 Bug Report

On Google Chrome Version 78.0.3904.87 (Official Build) (64-bit), and Selenium IDE version 3.14.0, when saving project, the project is saved as a ".txt" extension instead of ".side" one. Therefore you can't even import the project without changing the extension to ".side"

## To Reproduce

1. Make a new project
2. Try to save it

## Expected behavior

Project needs to be saved with a ".side" extension instead of ".txt"

## Project file reproducing this issue (highly encouraged)

/

## Environment

OS:  Windows 10
Selenium IDE Version:   3.14.0
Selenium SIDE Runner Version:   /
Node version:  /
Browser: Google Chrome
Browser Version:  78.0.3904.87 (Official Build) (64-bit)

**Comment by tourdedave** [comment]
This problem has popped up before (e.g., #790). Please try creating a new user profile in Chrome, installing the IDE, and saving the project to see if that resolves the issue.

**Comment by roland-jungwirth** [comment]
I can confirm the original issue on OSX.

@tourdedave as to your request:
Creating a new user, then installing the latest Selenium IDE, create a new project, then saving that resulted in the same issue (saves as .txt).

Even worse, Selenium IDE cannot read the txt file, resulting in the error

```
Error migrating project
Unknown file was received
```

### Environment

OS: Mac OSX
Browser: Google Chrome
Browser Version: Version 78.0.3904.87 (Official Build) (64-bit)

**Comment by tourdedave** [comment]
Thanks @roland-jungwirth for confirming. This also affects code export too.

While this annoying, the one nice thing is that the contents of the project file are intact. So the project can be imported after changing the file extension manually to `.side`.

I'll look into this today.

**Comment by roland-jungwirth** [comment]
@tourdedave, no problem. I've tried doing that (rename from *.txt to *.side) and then opening it but that's when I got the above mentioned error. I can't find an "import" function though - where is this?

**Comment by tourdedave** [comment]
You don't need to import (and there's no explicit menu to do an import). Just try opening the project after changing the extension to `.side`.

**Comment by roland-jungwirth** [comment]
Yes, I've done that but it fails. This is the error:

```
Error migrating project
Unknown file was received
```

I've done a cursory check of the file and it looks fine, so I'm assuming there is an issue with encoding? I have tested this with the attached file and can reproduce the error. The file was created by recording a random Google search.

[Test Google.txt](https://github.com/SeleniumHQ/selenium-ide/files/3818250/Test.Google.txt)

**Comment by tourdedave** [comment]
Renaming the file and opening it works for me.

Here's the renamed file: [Test.Google.side.zip](https://github.com/SeleniumHQ/selenium-ide/files/3818391/Test.Google.side.zip)

Can you try opening it?

**Comment by roland-jungwirth** [comment]
Yes, I can open your file.
But when I do the same on my machine (OSX), it doesn't work.

**Comment by corevo** [comment]
Make sure to rename it properly, adding .side in macos will simply yield test.side.txt

Cheers,
Tomer

**Comment by tourdedave** [comment]
It looks like Chrome now ignores the filename extension provided and sets one based on the MIME type instead. This issue does not happen in Firefox.

I'll get a fix in before the next release.

**Comment by roland-jungwirth** [comment]
@corevo thank you for stating the obvious - what a n00b error!

**Comment by tourdedave** [comment]
The fix has been released in `v3.15.0` which is available now from the Chrome store.
